apps/vanilla/src/pages/blackjack/Classes/dice-generator.ts

```diff
@@ -0,0 +1,22 @@
+import { Subscriber } from '../Types/subscriber';
+import { Publisher } from '../Types/publisher';
+
+import { PlayerTurnResult } from '../Types/player-turn-result';
+
+/** DiceGenerator .*/
```

Comment:
Don't just copy variable name. 
```suggestion
/** Dice generator .*/
```

I understand that this doc is pointless, but it's impossible to configure eslint to allow skipping "pointelss" documentation, eslint is not an AI 🫤